tests/functional/db_cleanup/test_db_cleanup.py

```diff
@@ -0,0 +1,159 @@
+#
+# -----------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -----------------------------------------------------------------------------
+
+import json
+import multiprocessing
+import os
+import unittest
+import time
+
+from codeCheckerDBAccess_v6.ttypes import *
+
+from libtest import codechecker
+from libtest import env
+
+
+class TestDbCleanup(unittest.TestCase):
+
+    def setUp(self):
+        self.test_workspace = os.environ['TEST_WORKSPACE']
+
+        test_class = self.__class__.__name__
+        print('Running ' + test_class + ' tests in ' + self.test_workspace)
+
+        self.codechecker_cfg = env.import_codechecker_cfg(self.test_workspace)
+        self.test_dir = os.path.join(self.test_workspace, 'test_files')
+
+        try:
+            os.makedirs(self.test_dir)
+        except os.error:
+            # Directory already exists.
+            pass
+
+    def _create_test_dir(self):
+        makefile = "all:\n\t$(CXX) -c a/main.cpp -o /dev/null\n"
+        project_info = {
+            "name": "hello",
+            "clean_cmd": "",
+            "build_cmd": "make"
+        }
+        source_main = """
+// Test file for db_cleanup
+#include "f.h"
+int main() { f(0); }
+"""
+        source_f = """
+// Test file for db_cleanup
+int f(int x) { return 1 / x; }
+"""
+
+        os.makedirs(os.path.join(self.test_dir, 'a'))
+
+        with open(os.path.join(self.test_dir, 'Makefile'), 'w') as f:
+            f.write(makefile)
+        with open(os.path.join(self.test_dir, 'project_info.json'), 'w') as f:
+            json.dump(project_info, f)
+        with open(os.path.join(self.test_dir, 'a', 'main.cpp'), 'w') as f:
+            f.write(source_main)
+        with open(os.path.join(self.test_dir, 'a', 'f.h'), 'w') as f:
+            f.write(source_f)
+
+    def _rename_project_dir(self):
```

Comment:
Is this a "private" method? Private methods in python are start with two underscores:
```py
def __rename_project_dir(self):
```